supporting_scripts/hyperion/consistency-check-benchmark/exercises.py

```
@@ -268,7 +303,8 @@ def create_exercise_variants(version: str, course: str, exercise: str, pecv_benc
         from cli.utils import ExerciseIdentifier
         logging.info("Successfully imported VariantManager and ExerciseIdentifier from pecv-bench.")
     except ImportError as e:
-        logging.error("ERROR: Failed to import VariantManager and ExerciseIdentifier from pecv-bench.")
+        logging.error(f"Step 4 failed: Failed to import VariantManager and ExerciseIdentifier from pecv-bench.")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)